src/game/PetAI.cpp

```diff
@@ -335,6 +335,16 @@ void PetAI::UpdateAI(const uint32 diff)
             return;
         }
 
+        if (!m_creature->HasInArc(2 * M_PI_F / 3, victim))
+        {
+	        m_creature->SetInFront(victim);
```

Comment:
Maybe you hit the space bar and didn't notice.

It is no biggie, you can do a fixup commit and then squash them and it'll be all good =)